metagpt/provider/bedrock_api.py

```diff
@@ -24,18 +25,19 @@ def __init__(self, config: LLMConfig):
         self.__client = self.__init_client("bedrock-runtime")
         self.__provider = get_provider(self.config.model)
         self.cost_manager = CostManager(token_costs=BEDROCK_TOKEN_COSTS)
-        if self.config.model in NOT_SUUPORT_STREAM_MODELS:
+        if self.config.model in NOT_SUPPORT_STREAM_MODELS:
             logger.warning(f"model {self.config.model} doesn't support streaming output!")
 
     def __init_client(self, service_name: Literal["bedrock-runtime", "bedrock"]):
         """initialize boto3 client"""
         # access key and secret key from https://us-east-1.console.aws.amazon.com/iam
-        self.__credentital_kwargs = {
-            "aws_secret_access_key": self.config.secret_key,
-            "aws_access_key_id": self.config.access_key,
-            "region_name": self.config.region_name,
+        self.__credential_kwargs = {
+            "aws_secret_access_key": os.environ.get("AWS_SECRET_ACCESS_KEY", self.config.secret_key),
+            "aws_access_key_id": os.environ.get("AWS_ACCESS_KEY_ID", self.config.access_key),
+            "aws_session_token": os.environ.get("AWS_SESSION_TOKEN", self.config.session_token),
```

Comment:
which scenario requires this session_token? It seems that access_key/key_id is enough?